string driver_id
string name
string car_brand
string car_type
string plate